src/Model/Export/Catalog/ProductType/ConfigurableDataProvider.php

```diff
@@ -17,6 +17,9 @@
 
 class ConfigurableDataProvider extends SimpleDataProvider
 {
+    /** @var string[] */
+    private const ALLOWED_TYPES = ['string', 'integer'];
```

Comment:
![image](https://user-images.githubusercontent.com/4511883/148724904-0c37fc41-ec5c-4415-8714-e15eb974d00c.png)
as we discussed before holidays, there are only strings or false value available here (excluding any customizations) so there is no need for introducing ALLOWED_TYPES